lib/tutorials/ko_KR/getting-started.md

```diff
@@ -18,114 +21,147 @@ _이 튜터리얼은 hapi v11.x.x와 호환됩니다._
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = Hapi.server({
+    port: 3000,
+    host: 'localhost'
+});
 
-server.start((err) => {
+const init = async () => {
 
-    if (err) {
-        throw err;
-    }
+    await server.start();
     console.log(`Server running at: ${server.info.uri}`);
+};
+
+process.on('unhandledRejection', (err) => {
+
+    console.log(err);
+    process.exit(1);
 });
+
+init();
 ```
 
-먼저 hapi가 필요합니다. 그런 다음 새로운 hapi 서버 객체를 만듭니다. 그다음에 서버에 연결을 추가하고 사용할 포트 번호를 전달합니다. 그리고 서버를 시작하고 서버가 실행 중이라는 로그를 남깁니다
+먼저 hapi가 필요합니다. 그런 다음 호스트(host)와 사용할 포트 번호를 포함한 설정 객체로 새로운 hapi 서버 객체를 만듭니다. 그 후에 서버를 시작하고 실행 중이라는 로그를 남깁니다.
 
-서버에 연결을 추가할 때, hostname, IP 주소 또는 Unix socket 파일, 서버에서 사용할 Windows named pipe을 제공할 수 있습니다. 자세한 정보는 [the API reference](/api/#serverconnectionoptions)를 봐주세요. 
+서버를 생성할 때 호스트 이름, IP 주소 또는 Unix 소켓 파일, 서버에서 사용할 Windows named pipe도 제공할 수 있습니다. 자세한 정보는 [the API reference](/api/#-server-options)를 봐주세요.
 
-## 경로(routes) 추가하기
+### 경로(routes) 추가하기
 
-하나 또는 두 개의 경로를 추가해야 실제로 동작하는 서버를 지금 가지고 있습니다. 어떻게 생겼는지 보겠습니다:
+하나 또는 두 개의 경로를 추가해야 실제로 동작하는 서버를 가지고 있습니다. 어떻게 생겼는지 보겠습니다.:
 
 ```javascript
 'use strict';
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = Hapi.server({
+    port: 3000,
+    host: 'localhost'
+});
 
 server.route({
     method: 'GET',
     path: '/',
-    handler: function (request, reply) {
-        reply('Hello, world!');
+    handler: (request, h) => {
+
+        return 'Hello, world!';
     }
 });
 
 server.route({
     method: 'GET',
     path: '/{name}',
-    handler: function (request, reply) {
-        reply('Hello, ' + encodeURIComponent(request.params.name) + '!');
+    handler: (request, h) => {
+
+        return 'Hello, ' + encodeURIComponent(request.params.name) + '!';
     }
 });
 
-server.start((err) => {
+const init = async () => {
 
-    if (err) {
-        throw err;
-    }
+    await server.start();
     console.log(`Server running at: ${server.info.uri}`);
+};
+
+process.on('unhandledRejection', (err) => {
+
+    console.log(err);
```

Comment:
I don't know if you got this code from the English version, but I find that this leads to the desired behavior (consistent with uncaught exceptions),
```js
process.on('unhandledRejection', (err) => {

    throw err;
});
```

Reply:
This code is same with code of original tutorial. 
I just translate original tutorial. So if original tutorial is changed, I will apply it.